src/Widgets/DataCountWidget/DataCountWidgetComponent.tsx

```diff
@@ -5,11 +5,45 @@ import {
 } from './DataCountWidgetClass'
 import { EditorNote } from '../../Components/EditorNote'
 
-provideComponent(DataCountWidget, ({ widget }) => (
-  <h1 className="h3 text-center">
-    <Count widget={widget} />
-  </h1>
-))
+provideComponent(DataCountWidget, ({ widget }) => {
+  const classNames: string[] = []
+
+  const style =
+    (widget.get('style') as
+      | 'display-1'
+      | 'display-2'
+      | 'display-3'
+      | 'display-4'
+      | 'display-5'
+      | 'display-6'
+      | 'h1'
+      | 'h2'
+      | 'h3'
+      | 'h4'
+      | 'h5'
+      | 'h6'
+      | 'text-small'
+      | 'body-font-size'
```

Comment:
* [ ] Since we are force casting anyway the types don't really mean much. I'd recommend sticking with the `string | null` we get thanks to `({ widget }: { widget: DataCountWidgetInstance })` and force cast (with a comment) where there is no easy way around it (`<Tag>`).

Reply:
I created an issue for that https://github.com/infopark/scrivito_js/issues/11596.